Atomic.md

```diff
@@ -0,0 +1,618 @@
+  TxnSignature: "3045022100EC5D367FAE2B461679AD446FBBE7BA260506579AF4ED5EFC3EC25F4DD1885B38022018C2327DB281743B12553C7A6DC0E45B07D3FC6983F261D7BCB474D89A0EC5B8"
+}
+```
+</details>
+
+#### 6.1.2. Sample Ledger
+
+<details open>
+<summary>
+This example shows what the ledger will look like after the transaction is confirmed.
+
+Note that the inner transactions are committed as normal transactions, and the `RawTransactions` field is not included in the validated version of the outer transaction.
+</summary>
+
+```typescript
+[
+  {
+    TransactionType: "Atomic",
+    Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+    Flags: "1",
+    TxnIDs: [
+      "7EB435C800D7DC10EAB2ADFDE02EE5667C0A63AA467F26F90FD4CBCD6903E15E",
+      "EAE6B33078075A7BA958434691B896CCA4F532D618438DE6DDC7E3FB7A4A0AAB"
+    ],
+    Sequence: 3,
+    Fee: "0",
+    SigningPubKey: "022D40673B44C82DEE1DDB8B9BB53DCCE4F97B27404DB850F068DD91D685E337EA",
+    TxnSignature: "3045022100EC5D367FAE2B461679AD446FBBE7BA260506579AF4ED5EFC3EC25F4DD1885B38022018C2327DB281743B12553C7A6DC0E45B07D3FC6983F261D7BCB474D89A0EC5B8"
+  },
+  {
+    TransactionType: "OfferCreate",
+    Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+    TakerGets: "6000000",
+    TakerPays: {
+      currency: "GKO",
+      issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+      value: "2"
+    },
+    AtomicTxn: {
+      Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+      OuterSequence: 3,
+      BatchIndex: 0
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  },
+  {
+    TransactionType: "Payment",
+    Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+    Destination: "rDEXfrontEnd23E44wKL3S6dj9FaXv",
+    Amount: "1000",
+    AtomicTxn: {
+      Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+      OuterSequence: 3,
+      BatchIndex: 1
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  }
+]
+```
+</details>
+
+### 6.2. Multiple Accounts
+
+In this example, two users are atomically swapping their tokens, XRP for GKO.
+
+#### 6.2.1. Sample Transaction
+
+<details open>
+<summary>
+
+The inner transactions are still not signed, but the `AtomicSigners` field is needed on the outer transaction, since there are two accounts' inner transactions in this `Atomic` transaction.
+</summary>
+
+```typescript
+{
+  TransactionType: "Atomic",
+  Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+  Flags: "1",
+  TxnIDs: [
+    "A2986564A970E2B206DC8CA22F54BB8D73585527864A4484A5B0C577B6F13C95",
+    "0C4316F7E7D909E11BB7DBE0EB897788835519E9950AE8E32F5182468361FE7E"
+  ],
+  RawTransactions: [
+    {
+      RawTransaction: {
+        TransactionType: "Payment",
+        Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+        Destination: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+        Amount: "6000000",
+        AtomicTxn: {
+          Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+          OuterSequence: 4,
+          Sequence: 10,
```

Comment:
This inner transaction is from the outer account, so it does not need a `Sequence`, or if it does, it should be 4, correct?

Reply:
I was picking random numbers and forgot to ensure they were the same 😅 
For `getSeqProxy` within the inner transaction, the `Sequence` value still needs to be here.